website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -39,6 +39,9 @@ Documents in this section go over steps to run different types of React Native r
 
 - Add and commit the extra file that got created at `sdks/hermes/.hermesversion`.
```

Comment:
headsup: can you update the paths to match the fact that we're in the monorepo now? ex. the bump-hermes-version script is here now: https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/hermes/bump-hermes-version.js

and the sdk file is here https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/sdks/.hermesversion

Reply:
Good point, updated 👍